packages/gitbook/src/lib/utils.ts

```diff
@@ -17,9 +17,9 @@ export function getContentTitle(
     // When we are rendering a site, always give priority to the customization title first
     // and then fallback to the site title
     if (parent?.object === 'site') {
-        return customization.title ?? parent.title ?? space.title;
+        return parent.title ?? space.title;
```

Comment:
Can you add a comment that the `parent.title` would already be overridden with the customization tittle while fetching the site